src/main/resources/META-INF/configurations/org.jahia.modules.jcontent.cfg

```diff
@@ -1,4 +1,3 @@
-# default configuration
```

Comment:
Hi, sorry didn't notice the first time, but it looks like we're potentially overriding existing (customer) changes/customizations with this one. Do we want to do this? or maybe just have this config as optional (enable only if we're overriding).